src/lib/plugins/addSubRows.ts

```diff
@@ -35,8 +35,17 @@ export const addSubRows =
 		NewTablePropSet<never>
 	> =>
 	() => {
-		const getChildren: ValidChildrenFn<Item> =
-			children instanceof Function ? children : (item) => item[children] as unknown as Item[];
+		let getChildren: ValidChildrenFn<Item>;
```

Comment:
Given that `ValidChildrenFn` has an optional return value, the original intention is that `getChildren` should return `undefined` if invalid.

Furtherfore, there is a semantic difference between returning `[]` and `undefined`, so the added `unknownVal.length > 0` check is an incorrect addition.